src/aat/resources/features/F-067/Befta_Case_Data_Extension.td.json

```diff
@@ -0,0 +1,9 @@
+{
+  "_guid_": "Befta_Case_Data_Extension",
+  "_extends_": "Befta_Default_Full_Case_Creation_Data",
+  "request": {
+    "body": {
+      "event_token": "${[scenarioContext][parentContext][childContexts][Befta_Default_Token_Creation_Data_For_Case_Creation][testData][actualResponse][body][token]}"
```

Comment:
The reason for adding this extension file was to override the token retrieval  from "Befta_Default_Token_Creation_Data_For_Case_Creation" instead of "Standard_Token_Creation_Data_For_Case_Creation"

Reply:
Ideally, this should be fixed in "Befta_Case_Creation_Base_Data" as a part of framework.